example-config.yml

```diff
@@ -11,6 +11,7 @@ raids:
       config:
         instance_identifier: unique-id-name
         database: test
+        region: us-east-1
```

Comment:
should this be something like `primary_region` for clarity?

Reply:
agreed, added with https://github.com/krumIO/raid-rds/pull/3/commits/914cab4a91685c5ba34d64cc689ce924934d2679